package/kernel/rtl88x2bu/Makefile

```diff
@@ -0,0 +1,53 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=rtl88x2bu
+PKG_RELEASE=1
+
+PKG_LICENSE:=GPLv2
+PKG_LICENSE_FILES:=
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/morrownr/88x2bu-20210702.git
+PKG_MIRROR_HASH:=d8bc277348cf97a1b645e2c1c07c9a0dcb9ce5796b7fd5bdce87d19b5f3e17bb
+PKG_SOURCE_DATE:=2022-10-19
+PKG_SOURCE_VERSION:=73a681a0db4a9ad132f85d567e225374c1e6a12b
+
+PKG_BUILD_PARALLEL:=1
+
+STAMP_CONFIGURED_DEPENDS := $(STAGING_DIR)/usr/include/mac80211-backport/backport/autoconf.h
+
+include $(INCLUDE_DIR)/kernel.mk
+include $(INCLUDE_DIR)/package.mk
+
+define KernelPackage/rtl88x2bu
+  SUBMENU:=Wireless Drivers
+  TITLE:=Driver for Realtek 88x2bu devices
+  DEPENDS:=+kmod-cfg80211 +kmod-usb-core +@DRIVER_11N_SUPPORT +@DRIVER_11AC_SUPPORT
+  FILES:=\
+	$(PKG_BUILD_DIR)/88x2bu.ko
+  AUTOLOAD:=$(call AutoProbe,88x2bu)
+  PROVIDES:=kmod-rtl88x2bu
```

Comment:
PROVIDES is not needed here. You only need it when multiple packages provide the same package.